app/controllers/account/billing-info/payment-info.js

```diff
@@ -0,0 +1,8 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
+  user: computed(function() {
+    return this.authManager.currentUser;
```

Comment:
Use it directly ... No computed property needed

Reply:
@niranjan94 I am now calling it directly using `user : this.authManager.currentUser` but it threw the following error 
```
payment-info.js:8 Uncaught (in promise) TypeError: Cannot read property 'authManager' of undefined
    at Module.callback (payment-info.js:8)
    at Module.exports (loader.js:106)
    at requireModule (loader.js:27)
    at Class._extractDefaultExport (index.js:394)
    at Class.resolveOther (index.js:114)
    at Class.superWrapper [as resolveOther] (utils.js:366)
    at Class.resolveController (globals-resolver.js:293)
    at Class.resolve (globals-resolver.js:140)
    at _resolve (container.js:1210)
    at Registry.resolve (container.js:749)
```
That's why I used a computed property :sweat_smile: .

Reply:
@uds5501 where exactly did you call it from ? Could you push the related code ?

Reply:
@niranjan94 I was calling this initially in `app/controllers/account/billing-info/payment-info.js` and then in the `app/components/forms/user-payment-info-form.js` . But now I am using intermediate element (made by a computed property) so I don't think this will be relevant